src/components/LanguageProvider/LanguageProvider.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode, FC, useMemo, useState } from 'react';
+import Languages from 'constants/languages';
+import LanguageContext from 'contexts/LanguageContext';
+import LanguageContextType from 'types/languageContextType';
+
+interface Props {
+  children: ReactNode;
+}
+
+const LanguageProvider: FC<Props> = function LanguageProvider({ children }) {
```

Comment:
Make it anonymous function expression. If the linter complains about it, disable a rule